setup.cfg

```diff
@@ -13,5 +13,6 @@ logging-level=INFO
 [flake8]
 max-line-length = 150
 max-complexity = 30
+ignore = E501
```

Comment:
Long URLs are ignored by standard. So I cannot see a reason why line length should be ignored.